### PR TITLE
Adds golden tools

### PR DESCRIPTION
**Pull request by praisenarsie**
:cl: Frozenguy5
add: Added golden tools (only admins can spawn them for now)
/:cl:

### BIG THANK YOU TO @wjohn FOR THE SPRITES

### CODE REVIEW

File: hippiestation/code/game/objects/items/tools/wrench.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/obj/item/wrench/gold
+	name = "golden wrench"
+	desc = "You gotta show your bling somehow!"
+	icon = 'hippiestation/icons/obj/tools.dmi'
+	icon_state = "goldwrench"
+	item_state = "goldwrench"
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/tools/wrench.dm:6]: you sure this works?

File: hippiestation/code/game/objects/items/tools/wirecutters.dm [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/obj/item/wirecutters/gold
+	name = "golden wirecutters"
+	desc = "You gotta show your bling somehow!"
+	icon = 'hippiestation/icons/obj/tools.dmi'
+	icon_state = "goldwirecutters"
+	item_state = "goldwirecutters"
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/tools/wirecutters.dm:6]: ?

File: hippiestation/code/game/objects/items/tools/weldingtool.dm [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/obj/item/weldingtool/gold
+	name = "golden welder"
+	desc = "You gotta show your bling somehow!"
+	icon = 'hippiestation/icons/obj/tools.dmi'
+	icon_state = "goldenwelder"
+	item_state = "goldenwelder"
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/tools/weldingtool.dm:6]: ?

File: hippiestation/code/game/objects/items/tools/screwdriver.dm [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/obj/item/screwdriver/gold
+	name = "golden screwdriver"
+	desc = "You gotta show your bling somehow!"
+	icon = 'hippiestation/icons/obj/tools.dmi'	
+	icon_state = "goldscrewdriver"
+	item_state = "goldscrewdriver"
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/tools/screwdriver.dm]: ?

File: hippiestation/code/game/objects/items/tools/wrench.dm (hunk 1 above)
Comment (praisenarsie) [reply hippiestation/code/game/objects/items/tools/wrench.dm:6]: yes

File: hippiestation/code/game/objects/items/tools/wirecutters.dm (hunk 2 above)
Comment (praisenarsie) [reply hippiestation/code/game/objects/items/tools/wirecutters.dm:6]: yes

File: hippiestation/code/game/objects/items/tools/weldingtool.dm (hunk 3 above)
Comment (praisenarsie) [reply hippiestation/code/game/objects/items/tools/weldingtool.dm:6]: yes

File: hippiestation/code/game/objects/items/tools/screwdriver.dm (hunk 4 above)
Comment (praisenarsie) [reply hippiestation/code/game/objects/items/tools/screwdriver.dm]: yes

**Review by yoyobatty — COMMENTED** [review]
Take pics !

**Comment by praisenarsie** [comment]
@yoyobatty look in the checks menu > icondiffbot

also i forgot to add inhands so give me a moment

**Comment by praisenarsie** [comment]
Added inhands, PR is ready now.

**Review by Carbonhell — DISMISSED** [review]

**Comment by Carbonhell** [comment]
+1

**Review by JohnGinnane — DISMISSED** [review]
these will inherit normal tool's toolspeed, is that intentional?

**Comment by praisenarsie** [comment]
@JohnGinnane i don't believe it does? i'll doublecheck now

**Comment by praisenarsie** [comment]
ok looks like malwarebytes blocked dreamdaemon but i'm sure it gives the right toolspeed

**Comment by Carbonhell** [comment]
Yeah it does, toolspeed 0 that will become 0.2 with the other PR. If you want to keep it to 0 specify it

**Comment by ghost** [comment]
yea sure when its ready +1

**Comment by ghost** [comment]
"...as my eyes grew accustomed to the light, details of the room within emerged slowly from the mist, strange animals, statues, and gold – everywhere the glint of gold."
– Howard Carter, The Discovery Of The Tomb Of Tutankhamen


+1

**Comment by earthmoonjune** [comment]
+1

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**